pkg/asset/machines/libvirt/machines.go

```diff
@@ -63,8 +63,8 @@ func provider(clusterID string, networkInterfaceAddress string, platform *libvir
 			APIVersion: "libvirtproviderconfig.openshift.io/v1beta1",
 			Kind:       "LibvirtMachineProviderConfig",
 		},
-		DomainMemory: 6144,
-		DomainVcpu:   4,
+		DomainMemory: providerMemory(platform),
```

Comment:
I think we should follow what other platforms do for the defaults like https://github.com/openshift/installer/blob/05388774c311c8dbf791c702cbd7678fd163739e/pkg/asset/machines/master.go#L136-L148

Reply:
done